stream_alert/shared/metrics.py

```diff
@@ -42,13 +42,13 @@
 except ValueError as err:
     ENABLE_METRICS = False
     LOGGER.error('Invalid value for metric toggling, expected 0 or 1: %s',
-                 err.message)
+                 str(err))
```

Comment:
I expect it was written this way because without the cast the unit test fails with the most infuriating message: ```Expected: error('Invalid value for metric toggling, expected 0 or 1: %s', ValueError("invalid literal for int() with base 10: 'bad'"))                                    
Actual: error('Invalid value for metric toggling, expected 0 or 1: %s', ValueError("invalid literal for int() with base 10: 'bad'"))```

Reply:
wut

Reply:
oh.. change the unit test to `from mock import ANY` and use `ANY` as the expected item for `err`..

Reply:
that said you can also omit the `err.message` and pass `err` directly